packages/forms/src/styled/tiles/StyledTileIcon.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { DEFAULT_THEME, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import { math } from 'polished';
+
+const COMPONENT_ID = 'forms.tile_icon';
+
+interface IStyledTileIconProps {
+  isStacked?: boolean;
+}
+
+const sizeStyles = (props: IStyledTileIconProps & ThemeProps<DefaultTheme>) => {
+  const iconSize = math(`${props.theme.iconSizes.md} * 2`);
+
+  return css`
+    & > * {
+      width: ${iconSize};
+      height: ${iconSize};
+    }
+  `;
+};
+
+export const StyledTileIcon = styled.span.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledTileIconProps>`
+  display: block;
+  position: ${props => props.isStacked && 'absolute'};
+  top: ${props => props.theme.space.base * 6}px;
+  text-align: center;
+  line-height: 0;
+  ${props => (props.theme.rtl ? 'right' : 'left')}: ${props => props.theme.space.base * 5}px;
```

Comment:
Neither `right/left` or `top` is applicable for non-stacked tiles. Feels like the absolute positioning could all be moved to `sizeStyles` and simplified by using local variables.

Reply:
I've moved this logic to the existing `sizeStyles` utility to help clarify.